core/assets/bundles/bundle.properties

```diff
@@ -892,6 +892,7 @@ block.coal-centrifuge.name = Coal Centrifuge
 block.power-node.name = Power Node
 block.power-node-large.name = Large Power Node
 block.surge-tower.name = Surge Tower
+block.diode.name = Diode
```

Comment:
Maybe call it "Battery Diode" so people don't assume it works for graphs without batteries. Not sure about that, though.

Reply:
I thought the fact that it only worked with batteries was a bug?

Reply:
No, it's a current limitation of how the diode works. Unless a different method of directional power transfer is used, it'll stay that way.